### ISSUE TITLE
Panini.refresh() does not update partials when only line order is changed.

### ISSUE THREAD

**Issue by DamianPereira** [zurb/panini#115]
Hi, I have a header.html with a dropdown menu, which looks like this:

```
<header class="nav-bar row align-middle">
  <div class="nav-bar__logo small-3 columns">
    <img src="{{root}}assets/img/logo.png">
  </div>
  <div class="nav-bar__menu columns">
    <ul class="dropdown menu align-right" data-dropdown-menu>
      <li><a class="nav-bar__link" href="{{root}}contacto.html">Contacto</a></li>
      <li><a class="nav-bar__link" href="{{root}}nosotros.html">Nosotros</a></li>
      <li><a class="nav-bar__link" href="{{root}}particulares.html">Particulares</a></li>
      <li><a class="nav-bar__link" href="{{root}}empresas.html">Empresas</a></li>
      <li><a class="nav-bar__link" href="{{root}}index.html">Inicio</a></li>
    </ul>
  </div>
</header>
```
This file is included in the default layout, with {{> header}}.

If I change the order of the menu items, moving the menu items around, the final page is not updated, but if I change a character it works as expected. The refresh function is called by Gulp, but it does not correctly refresh the changes in the partial.

**Comment by gakimball** [comment]
Is this the [ZURB Template](https://github.com/zurb/foundation-zurb-template) you're using?

**Comment by DamianPereira** [comment]
Yes, it's a new project using the foundation-cli install of the zurb template, only added layouts, pages, and sass files.

**Comment by kaluogh** [comment]
//gulp 4.0.2 panini 1.6.3
gulp.watch('src/html/pages/**/*').on('change', gulp.series('compile-html', browserSync.reload));
    gulp.watch('src/html/{layouts,includes,helpers,data}/**/*').on('change', gulp.series(async () => { await panini.refresh() }, 'compile-html', browserSync.reload));

**Comment by kaluogh** [comment]
A pages change ...
B template change
   1 panini refresh
   2 rebuild html return gulp.src(xxx).pipe(panini(paniniOption)).pipe(gulp.dest(xxxx));
   3 browserSync.reload
